pallets/route-executor/src/lib.rs

```diff
@@ -739,6 +642,8 @@ impl<T: Config> Pallet<T> {
 		Ok(amount_out)
 	}
 
+	//TODO: this and all other calculation wont be needed, also in benchmark, remove it
```

Comment:
i believe this is still used by add_route mechanic

Reply:
true, used in DCA sell too